src/platforms/esp32/test/main/test_erl_sources/test_list_to_binary.erl

```diff
@@ -0,0 +1,34 @@
+%
+% This file is part of AtomVM.
+%
+% Copyright 2023 Fred Dushin <fred@dushin.net>
+%
+% Licensed under the Apache License, Version 2.0 (the "License");
+% you may not use this file except in compliance with the License.
+% You may obtain a copy of the License at
+%
+%    http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS,
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+% See the License for the specific language governing permissions and
+% limitations under the License.
+%
+% SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+%
+
+-module(test_list_to_binary).
+
+-export([start/0]).
+
+start() ->
+    ok = test_empty_list_to_binary(),
+    ok.
+
+test_empty_list_to_binary() ->
+    <<"">> = erlang:list_to_binary(id([])),
```

Comment:
I personally prefer <<>> for empty binaries, they look emptier.

Do we want to settle on a style guide for these? ;-)

Reply:
Surely there must be some esoteric escape sequence (`^H`) that would be better?  ;)